src/LaunchDarkly.Client/LaunchDarkly.Client.nuspec

```diff
@@ -14,5 +14,8 @@
     <releaseNotes>Switch to asynchronous HTTP client</releaseNotes>
     <copyright>$copyright$</copyright>
     <tags>Feature Configuration</tags>
+     <dependencies>
+      <dependency id="Newtonsoft.Json" version="8.0.3" />
```

Comment:
I think what's still lacking for me is an understanding of what this does-- good that it fixed the problem, but did this ever work without this dependency explicitly stated?